app/index.js

```diff
@@ -63,6 +63,7 @@ const flash = require('../js/flash')
 const contentSettings = require('../js/state/contentSettings')
 const privacy = require('../js/state/privacy')
 const basicAuth = require('./browser/basicAuth')
+const async = require('async')
```

Comment:
It's better to require only `async/queue` so it doesn't pull in everything, pls do in a follow up, I won't block merging this though on it.